messaging-experiments.json

```diff
@@ -1116,5 +1116,279 @@
         }
       ]
     }
+  },
+  {
+    "id": "MULTI-STAGE-ABOUTWELCOME-V2-79-RELEASE",
+    "enabled": true,
+    "filter_expression": "env.version >= '79.' && (env.channel == 'default' || env.channel == 'beta')",
+    "targeting": "localeLanguageCode == \"en\" && [userId, \"message-multi-stage-aboutwelcome-v2-79\"]|bucketSample(0, 2000, 10000) && ((isFirstStartup && !('trailhead.firstrun.didSeeAboutWelcome'|preferenceValue)) || 'message-multi-stage-aboutwelcome-v2-79' in activeExperiments)",
+    "arguments": {
+      "slug": "message-multi-stage-aboutwelcome-v2-79",
+      "userFacingName": "Multi-Stage About:Welcome V2 79 Release",
+      "experimentDocumentUrl": "https://experimenter.services.mozilla.com/experiments/multi-stage-aboutwelcome-v2-79-release/",
+      "userFacingDescription": "Study to validate Multi-stage onboarding experience.",
+      "isEnrollmentPaused": true,
+      "branches": [
+        {
+          "slug": "control",
+          "ratio": 1,
+          "groups": [
+            "aboutwelcome"
+          ],
+          "value": {}
+        },
+        {
+          "slug": "treatment-a",
+          "ratio": 1,
+          "groups": [
+            "aboutwelcome"
+          ],
+          "value": {
+            "id": "multi-stage-welcome-import",
+            "template": "multistage",
+            "screens": [
+              {
+                "id": "AW_GET_STARTED",
+                "order": 0,
+                "content": {
+                  "zap": true,
+                  "title": {
+                    "string_id": "onboarding-welcome-header"
+                  },
+                  "subtitle": "The fast, safe, and private browser that's backed by a non-profit.",
+                  "primary_button": {
+                    "label": "Start Setup",
+                    "action": {
+                      "navigate": true
+                    }
+                  },
+                  "secondary_button": {
+                    "text": "Have an account?",
+                    "label": "Sign in",
+                    "position": "top",
+                    "action": {
+                      "type": "OPEN_URL",
+                      "addFlowParams": true,
+                      "data": {
+                        "args": "https://accounts.firefox.com/?service=sync&action=email&context=fx_desktop_v3&entrypoint=activity-stream-firstrun&style=trailhead",
+                        "where": "current"
+                      }
+                    }
+                  }
+                }
+              },
+              {
+                "id": "AW_IMPORT_SETTINGS",
+                "order": 1,
+                "content": {
+                  "zap": true,
+                  "title": "Import your passwords, bookmarks,\u00a0and more",
```

Comment:
Interesting, the YAML version doesn't show the unicode non-breaking space. Can you double check?

Reply:
yes, it do show up in json, so all good